From 1cb7b3ece5427163e6512677e83007eda6e46ba7 Mon Sep 17 00:00:00 2001
From: YouMin Chen <cym@rock-chips.com>
Date: Thu, 21 Jun 2018 14:30:26 +0800
Subject: [PATCH] arm64: dts: rockchip: px30: disable ddr standby

The function of ddr standby influence memory stability,
so disable it for the present.

Change-Id: I9df7381944687da8819665009b162c387eb06ceb
Signed-off-by: YouMin Chen <cym@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/px30-dram-default-timing.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/px30-dram-default-timing.dtsi b/arch/arm64/boot/dts/rockchip/px30-dram-default-timing.dtsi
index f429cc8669ab..c75c5ef4ef2a 100644
--- a/arch/arm64/boot/dts/rockchip/px30-dram-default-timing.dtsi
+++ b/arch/arm64/boot/dts/rockchip/px30-dram-default-timing.dtsi
@@ -16,8 +16,8 @@
 		pd_idle = <13>;
 		sr_idle = <93>;
 		sr_mc_gate_idle = <0>;
-		srpd_lite_idle	= <0>;
-		standby_idle = <4000>;
+		srpd_lite_idle = <0>;
+		standby_idle = <0>;
 
 		auto_pd_dis_freq = <1066>;
 		auto_sr_dis_freq = <800>;
-- 
2.35.3

